modules/tinymce/changelog.txt

```diff
@@ -18,6 +18,8 @@ Version 5.7.0 (TBD)
     Fixed the "Dropped file type is not supported" notification incorrectly showing when using an inline editor #TINY-6834
     Fixed an issue with external styles bleeding into TinyMCE #TINY-6735
     Fixed incorrect return types on `editor.selection.moveToBookmark` #TINY-6504
+    Fixed the type signature for `editor.selection.getRng` incorrectly returning null #TINY-6843
+    Fixed the type signature for `editor.selection.setCursorLocation` incorrectly allowing a node with no offset #TINY-6843
```

Comment:
Nitpicks, so if you disagree - no issues. I think `null` should be in backticks though

```suggestion
    Fixed the type signature for `editor.selection.getRng()` incorrectly returning `null` #TINY-6843
    Fixed the type signature for `editor.selection.setCursorLocation()` incorrectly allowing a node with no `offset` #TINY-6843
```

Reply:
Agreed, as those are keywords.